app/views/step_by_step_pages/index.html.erb

```diff
@@ -132,7 +132,7 @@
               <td class="govuk-table__cell">
                 <% if step_by_step_page.has_draft? %>
                   <%= link_to "/#{step_by_step_page.slug}", preview_url(step_by_step_page.slug, auth_bypass_id: step_by_step_page.auth_bypass_id), target: "_blank" %>
-                <% elsif step_by_step_page.status == "live" %>
+                <% elsif step_by_step_page.status == "published" %>
                   <%= link_to "/#{step_by_step_page.slug}", live_url(step_by_step_page.slug), target: "_blank" %>
```

Comment:
What about changing `live_url` to `public_url`? as `live` is being purged from the codebase.

Reply:
Updated and fixed-up.